Themes/default/css/responsive.css

```diff
@@ -69,7 +69,8 @@
 }
 
 @media screen and (min-width: 721px) {
-	#mobile_action {
+	#mobile_action,
+	#mobile_moderation {
 		display: none !important;
```

Comment:
Are we able to nuke these `!important` priority overrides? Or will that result in death and despair?

Reply:
I’m afraid it will break, it has a strange design so we should probably leave it for later.